web/themes/joinup/prototype/sass/components/_form-table.sass

```diff
@@ -36,13 +44,53 @@ thead th.form-table__header
 
 // Dependencies
 // --------------------------------------------------
-.form-table__row
-  &.draggable
-    .draggable__icon, .form-managed-file
-      display: table-cell
+.form-table
+  .form-table__row
+    .dropbutton-multiple
+      min-height: auto
+      margin: 0
+      padding: 0
+    .dropbutton
+      display: flex
+    .dropbutton-widget
+      position: static
+    .dropbutton-toggle
+      display: none
+      visibility: none
```

Comment:
Done. It was duplicate, `display: none` left to avoid additional space leftovers.